webpack.config.shared.js

```diff
@@ -13,35 +13,44 @@ module.exports = {
       'zendesk_widget': './static/js/entry/zendesk_widget.js',
     },
     module: {
-      loaders: [
+      rules: [
         {
           test: /\.(svg|ttf|woff|woff2|eot|gif)$/,
-          loader: 'url-loader'
+          use: 'url-loader'
         },
         {
           test: /\.scss$/,
           exclude: /node_modules/,
-          loader: 'style!css!postcss!sass'
+          use: [
+            { loader: 'style-loader' },
+            { loader: 'css-loader' },
+            { loader: 'postcss-loader' },
+            { loader: 'sass-loader' },
+          ]
         },
         {
           test: /\.css$/,
           exclude: /node_modules/,
-          loader: 'style!css'
+          use: [
+            { loader: 'style-loader' },
+            { loader: 'css-loader' }
+          ]
         },
       ]
     },
-
     resolve: {
-      modulesDirectories: ['node_modules'],
-      extensions: ['', '.js', '.jsx'],
+      modules: [
+        path.join(__dirname, "static/js"),
+        "node_modules"
+      ],
+      extensions: ['.js', '.jsx'],
       alias: {
         react: path.resolve('./node_modules/react')
       }
     }
   },
-
   babelSharedLoader: {
-    text: /\.jsx?$/,
+    test: /\.jsx?$/,
```

Comment:
Yeah I guess from the documentation that it basically whitelists the files it processes, but we don't really have files that aren't of this file type right now anyway